app/models/user.rb

```diff
@@ -1,2 +1,9 @@
 class User < ApplicationRecord
+  has_many :likes, foreign_key: 'author_id'
+  has_many :comments, foreign_key: 'author_id'
+  has_many :posts, foreign_key: 'author_id'
+
+  def most_recent_posts
+    Post.order(created_at: :desc).includes(:author).limit(5)
+  end
```

Comment:
- [ ] Please replace `Post` with `posts` because we want to return the posts belonging to this user.